### PR TITLE
[New] Add colresizable w/ git auto-update

### PR DESCRIPTION
**Pull request by sashberd**
Pull request for issue: #10700
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/alvaro-prieto/colResizable
 * Official website (optional, not the repository): http://bacubacu.com/colresizable/
 * NPM package url (optional): https://libraries.io/npm/colresizable
 * License and its reference: MIT&GPL - 2.0
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 17
   - Count of stars: 228
   - Count of forks: 134
 * NPM download stats (optional):
   - Downloads in the last day: 70
   - Downloads in the last week: 448
   - Downloads in the last month: 1582

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] Doesn't already exist / has duplicate issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [ ] Auto-update target/source is valid.
 * [ ] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
 * [x] The parent of the commit(s) in the PR is not older than 3 days.
 * [x] Pull request is sent from a non-master branch with a meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sashberd** [comment]
MEMO!!!
ON git there is v1.6.0 and v1.6 from my review they are the same. The issue with [question ](https://github.com/alvaro-prieto/colResizable/issues/89)was opened to author. If they are the same the 3rd commit should add .gitigonre with one of the not needed version, otherwise we need to host both of them

File: ajax/libs/colresizable/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+  "name": "colresizable",
+  "filename": "colResizable-1.6.min.js",
```
Comment (extend1994) [inline ajax/libs/colresizable/package.json]: Based on the above reason, please rename `colResizable-1.6.min.js` to `colResizable.min.js` and update `filename` accordingly.

File: ajax/libs/colresizable/package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  },
+  "keywords": [
+    "resize table",
+    "resize columns",
+    "resize",
+    "resizable",
+    "columns",
+    "jquery-plugin",
+    "table",
+    "range slider",
```
Comment (extend1994) [inline ajax/libs/colresizable/package.json]: Let's update this to only `slider` and remove other keywords with spaces
The reason is, for example, 
when you type `resize table`, it matched `resize` and `table`, so there is no need to use space in the `keywords`.

Comment (sashberd) [reply ajax/libs/colresizable/package.json]: @extend1994 So what should I do with autoupdate part of package.json? Should I remove it temporary until we will get answer from author about next releases?

**Comment by sashberd** [comment]
@extend1994 I updated the PR. If other parts are OK, just give me to know about autoupdate part in package.json

**Comment by sashberd** [comment]
@xtuc What about this one. Here we had some issues with file names and .gitignore excludes
Would you like me to remove all version and leave only package.json or merge it as it was approved?

**Comment by xtuc** [comment]
@sashberd that would be great, we don't need gitignore here I think

**Comment by sashberd** [comment]
@xtuc As you can see from my [conversation](https://github.com/alvaro-prieto/colResizable/issues/89) with lib author, there are indeed files with diff names, but with the same content. That is why we add those files to .gitigone

File: ajax/libs/colresizable/package.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        "files": [
+          "colResizable-*.+(js|map)"
+        ]
+      },
+      {
+        "basePath": "",
+        "files": [
+          "source/colResizable-*.+(js|map)"
+        ]
+      }
```
Comment (MattIPv4) [inline ajax/libs/colresizable/package.json]: This can be combined into a single filemap, please :)

File: .gitignore [hunk 4]
```diff
@@ -333,4 +334,4 @@ node_modules/
 /ajax/libs/*/*/**/custom
 /ajax/libs/*/*/**/README
 /ajax/libs/*/*/**/README.md
-/ajax/libs/*/*/**/README.markdown
+/ajax/libs/*/*/**/README.markdown
```
Comment (MattIPv4) [inline .gitignore]:
```suggestion
/ajax/libs/*/*/**/README.markdown

```
Please preserve newline :)

**Pull request merged**